.github/workflows/go.yml

```diff
@@ -134,6 +134,7 @@ jobs:
         TSDBOSS: ${{ matrix.test-setups.tsdboss }}
         SHORTNAME: ${{ matrix.test-setups.shortname }}
       run: go test -race ./pkg/tests/end_to_end_tests/ -use-extension=$EXT -use-timescaledb=$TSDB -use-timescale2=$TSDB2 -use-timescaledb-oss=$TSDBOSS -use-multinode=$MULTI -postgres-version-major=$PG | tee $SHORTNAME-test-run.log  2>&1
+      shell: bash --noprofile --norc -eo pipefail {0}
```

Comment:
I think we should use the defaults to set the shell to this for all steps of the workflow.
```
defaults:
  run:
    shell: bash --noprofile --norc -eo pipefail {0}
```
https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#defaults